nodes/For.py

```diff
@@ -1,2 +1,25 @@
+"""
+For astroid node
+
+A for loop.
+
+Attributes:
+    - target  (Node(Name | Tuple | List))
+        - Holds the variable(s) the loop assigns to as a single node.
+    - iter    (Node)
+        - The node to be looped over.
+    - body    (Node)
```

Comment:
I'm quite confused about how to know the type of the attribute by looking at the source code. But I think if the type of the attribute is a list, it will be presented in a source code like self.items = [] in the __init__ function of that node, like the one in dict. So I assume that self.body = body just means that body is a Node type instead of a list??? Also I don't know what Stmt is Q__Q.